perf/prometheus-install/prometheus-scrape-configs.yaml

```diff
@@ -0,0 +1,286 @@
+
+- job_name: 'istio-mesh'
+  kubernetes_sd_configs:
+  - role: endpoints
+    namespaces:
+      names:
+      - istio-system
+
```

Comment:
Is this same or different than the stock prom config?

Reply:
If by stock, you mean the istio chart, this is the same namespace, although the service name does change from just prometheus to istio-prometheus (this is unavoidable).